JenkinsfileRT

```diff
@@ -0,0 +1,74 @@
+//@Library('utils@master') _  // For development only.
+
+if (utils.scm_checkout()) return
+
+def test_env = [
+    "HOME=./",
+    "PATH=./miniconda/bin:/bin:/usr/bin",
+    "TEST_BIGDATA=/data4/jwst_test_data",
+    "CRDS_SERVER_URL=https://jwst-crds.stsci.edu",
+    "CRDS_CONTEXT=jwst_0445.pmap",
+    "CRDS_PATH=./test_outputs/crds_cache",
+]
+
+//def python_version = ['3.5', '3.6']
+//def numpy_version = ['1.13', '1.14']
+
+def CONDA_DEPS = "asdf \
+                  astropy \
+                  crds \
+                  dask \
+                  drizzle \
+                  fitsblender \
+                  flake8 \
+                  gwcs \
+                  jsonschema \
+                  jplephem \
+                  matplotlib \
+                  namedlist \
+                  numpy \
+                  photutils \
+                  scipy \
+                  six \
+                  spherical-geometry \
+                  stsci.image \
+                  stsci.imagestats \
+                  stsci.stimage \
+                  stsci.tools \
+                  verhawk"
+def CONDA_CHANNEL = "http://ssb.stsci.edu/conda-dev"
```

Comment:
There's really nothing stopping anyone from using the public channel, rather than dev, then filling in the blanks with `pip install -q [git+https://github.com/some/repo.git#egg=name]` for each bleeding edge package they require.

Unfortunately time will be lost compiling the dependencies every single time this runs.

Reply:
> each bleeding edge package they require

Hopefully not that much? Usually it is just Astropy, right? I mean, you can't possibly deliver dev of everything to DMS, can you? I just thought that testing what you actually want to deliver might be more useful.

Reply:
DMS receives `master` for almost every dependency of the pipeline right now. That's going to change eventually, however at the moment you can't run the pipeline with the stable/public releases of most packages. `jwst` inevitably requires  _[unreleased Astropy feature here]_ used by _[development code in another package there]_ so distributing that dependency to the public is pretty difficult. 

I'd like to figure out exactly which dependencies are doing this. ASDF tracks changes made upstream in Astropy very closely, and maybe the same can be said for WCS-related packages... As for others I'm not 100% sure. (< This sounds like it needs to be in its own issue.)